# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-ia64-cpufreq.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./drivers/cpufreq/ia64-acpi-cpufreq.c.vanilla	2024-10-05 22:04:16.056620179 +0200
+++ ./drivers/cpufreq/ia64-acpi-cpufreq.c	2024-10-05 22:04:24.133286671 +0200
@@ -302,7 +302,7 @@
 }
 
 
-static int
+static void
 acpi_cpufreq_cpu_exit (
 	struct cpufreq_policy   *policy)
 {
@@ -316,8 +316,6 @@
 		kfree(policy->freq_table);
 		kfree(data);
 	}
-
-	return (0);
 }
 
 
